### PR TITLE
Update Styles: Full Set modals

### LINKED ISSUE #1393: Implement design for buy / sell full set modals

### PR DESCRIPTION
**Pull request by robhyrk**
closes #1393

### CODE REVIEW

File: components/markets/BuyFullSetForm.tsx [hunk 1]
```diff
@@ -26,31 +29,34 @@ const BuyFullSetForm = ({
...
   const [maxTokenSet, setMaxTokenSet] = useState<Decimal>(new Decimal(0));
 
-  const { data: baseAssetBalance } = useBalance(
-    wallet.getActiveSigner()?.address,
-    baseAssetId,
-  );
+  const extrinsicBase = wallet.activeAccount?.address
+    ? sdk
+        ?.asRpc()
+        .api.tx.balances.transfer(wallet.activeAccount?.address, ZTG.toFixed(0))
```
Comment (yornaath) [inline components/markets/BuyFullSetForm.tsx:46]: @Robiquet When doing transfers using a proxy should we transfer on behalf of `wallet.readlAddress` ?

File: components/markets/BuyFullSetForm.tsx [hunk 2]
```diff
@@ -26,31 +29,34 @@ const BuyFullSetForm = ({
...
+    : undefined;
+  const { data: fee } = useExtrinsicFee(extrinsicBase);
 
   const { data: balances } = useAccountPoolAssetBalances(
     wallet.getActiveSigner()?.address,
     pool,
   );
 
+  const { data: baseAssetBalance } = useBalance(
+    wallet.getActiveSigner()?.address,
```
Comment (yornaath) [inline components/markets/BuyFullSetForm.tsx:56]: Same as above, should we use `wallet.realAddress`? @Robiquet

File: components/markets/SellFullSetForm.tsx [hunk 3]
```diff
@@ -47,9 +45,12 @@ const SellFullSetForm = ({
   const [amount, setAmount] = useState<string>("0");
   const [maxTokenSet, setMaxTokenSet] = useState<Decimal>(new Decimal(0));
 
-  const colors = market?.categories
-    ? calcMarketColors(marketId, market.categories.length)
-    : [];
+  const extrinsicBase = wallet.activeAccount?.address
+    ? sdk
+        ?.asRpc()
+        .api.tx.balances.transfer(wallet.activeAccount?.address, ZTG.toFixed(0))
```
Comment (yornaath) [inline components/markets/SellFullSetForm.tsx:51]: @Robiquet Same as the other comments, should we transfer on behalf of `wallet.readAddress` when using proxies?

**Review by yornaath — CHANGES_REQUESTED** [review]
Might need some changes to which address is used when creation extrinsics. Ping @Robiquet

**Review by Robiquet — APPROVED** [review]

**Comment by Robiquet** [comment]
Don't think we can merge this without approval from @yornaath. If we don't hear back close and reopen this then I'll approve

**Pull request closed without merging**